Content.Server/Body/Systems/MetabolizerSystem.cs

```diff
@@ -133,13 +133,24 @@ private void TryMetabolize(Entity<MetabolizerComponent, OrganComponent?, Solutio
                 return;
             }
 
+            // Copy the solution do not edit the original solution list
+            var list = solution.Contents.ToList();
+
+            // excluding chemicals which bloodstream declares as blood
+            var ev = new MetabolismExclusionEvent(list);
+            RaiseLocalEvent(solutionEntityUid.Value, ref ev);
```

Comment:
i'm not sure how I feel about this event given that it just outright lets any subscriber modify the entire list of what the metabolizer thinks it has instead of gathering a set to ignore